src/style.css

```
@@ -1 +1,79 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+body {
+  width: 100%;
+  background: #e5e5e5;
+  margin: 0;
+}
+
+.header {
+  justify-content: space-between;
+  background-color: #fff;
+  display: flex;
+  align-items: center;
+  font-family: "Roboto", sans-serif;
+  padding: 0 50px;
+}
+
+.logo {
+  font-size: 0;
+}
+
+.nav {
+  margin: 0;
+}
+
+.nav__list {
+  display: flex;
+  text-transform: uppercase;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  list-style: none;
+  text-decoration: none;
+  text-align: center;
+  margin-left: 20px;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.nav__link {
+  position: relative;
+  padding: 23px 0;
+  cursor: pointer;
+  color: #000;
+  text-decoration: none;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 60px;
+}
+
+.nav__link::after {
+  display: block;
+  content: "";
+  background: #00acdc;
+  border-radius: 8px;
+  transition: 300ms;
```

it's not a good practice not to set `transition-property`.
So we can specify what to change exactly
```suggestion
  transition: color 300ms;
```